styles/style.css

```diff
@@ -0,0 +1,571 @@
+
+  .nav__hamb_none,
+  .nav__home_mob_none { display: none; }
+
+  .nav__home_desk { display: initial; }
+
+  .nav__link_col,
+  .nav__signin_col,
+  .nav__cart_col,
+  .nav__lang_col { height: unset; }
+
+  .nav__checkbox:checked ~ .nav__items { display: flex; }
+
+  .nav {
+    display: flex;
+    justify-content: space-between;
+    height: calc(14.05rem - 9.3vw);
+    min-height: 9.66em;
+  }
+  
+  .nav__items {
+    display: flex;
+    justify-content: inherit;
+  }
+
+  .nav__home_desk {
+    display: flex;
+    align-items: center;
+  }
+  
+  .nav__items_l { flex-basis: calc(23.34rem + 6.57vw); }
+
+  .nav__links {
+    display: inherit;
+    justify-content: space-around;
+    width: 23rem;
+  }
+
+  .nav__link { flex-grow: 1; }
+  
+  .nav__items_r {
+    /* flex-basis: 29.9em;
+    flex-basis: 32em; */
+    flex-basis: calc(15.17em + 17.26vw);
+  }
+
+  .nav__search {
+    border-bottom: 1px solid #000;
+    align-self: center;
+  }
+
+  .search__bar { border: none; }
+  
+  .nav__cart {
+    display: inline-flex;
+    align-items: center;
+  }
+
+  .nav__flag { width: calc(.278rem + 1.328vw); }
+
+  .header {
+    height: calc(3.887rem + 18.616vw);
+    /* grid-template-rows: 7.3em 8.6em; */
+    grid-template-rows: calc(8.22vw - .71em) calc(3.8em + 4.93vw);
+    background: url(../logos/rob-potter-422534-unsplash.png) no-repeat 0/cover;
+  }
+
+  .product {
+    height: calc(57.95em - 27.12vw);
+    grid-template-rows: minmax(4.2em, calc(12.69em - 8.71vw)) minmax(3.3em, calc(8.91em - 5.75vw));
+    min-height: 31.5em;
+  }
+  
+  .product__items {
+    justify-content: space-between;
+    padding: 0 1em;
+  }
+  
+  .product__item {
+    height: calc(14.59em + 4.93vw);
+    width: calc(3.58em + 13.97vw);
+    max-height: 19.4em;
+    /* max-width: 17.2em; */
+  }
+
+  .product__link {
+    /* grid-template-rows: 16.8em 1.5em; */
+    grid-template-rows: calc(11.99em + 4.93vw) 1.5em;
+  }
+
+  .product__img {
+    max-width: 15.7em;
+    max-height: 14.5em;
+    width: calc(6.08em + 9.86vw);
+    height: calc(6.49em + 8.22vw);
+  }
+
+  .product__clr {
+    width: .58em;
+    height: .58em;
+  }
+
+  .product__radio:checked + .product__clr {
+    width: .75em;
+    height: .75em;
+  }
+
+  .product__clrs {
+    /* width: 4.5em; */
+    width: calc(1.29em + 3.29vw);
+  }
+
+  .product__item:nth-child(3),
+  .product__item:nth-child(4) { display: grid; }
+
+  .product__item::before {
+    background: #000;
+    opacity: 0.2;
+    content: "";
+    width: .1em;
+    height: 8.4em;
+    position: absolute;
+    /* margin: 2.8em 17.7em;
+    margin: 3.5em 19.7em; */
+    margin: calc(5.75vw - 2.11em) calc(3.76em + 16.44vw);
+  }
+
+  .product__item_4::before { display: none; }
+
+  .product__item_1 .product__clrs { flex-direction: row-reverse; }
+  
+  .product__item_1 .product__clrs .product__clr_yl { order: -1; }
+}
+
+@media only screen and (min-width: 1170px) {
+  .product__link { grid-template-rows: 16.8em 1.5em; }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887702)